wasm/v8/benchmark.md

```diff
@@ -61,3 +65,19 @@ for 10k blocks this translates to (approximately) 4.73 minutes for the rust vers
 This time is different when running on docker since we're in production mode and have access to the cache. When using the cache the difference between the two is negligible but when un-cached the difference is very noticeable.
 
 On a clean instance you can see a 50-70% slower execution on the JS side than the rust-wasm execution, no cache, for a 1000 blocs. This trend is also true for a number of blocks n that converge to 10k blocks.
+
+### Polyfill + Bundle Loading Benchmark
+
+Benchmark comparing two strategies for loading JS scripts on a fresh context:
```

Comment:
Didn't though of telling you that the Go has a tool `benchstat` that can easily compare two benchmarks and note the difference: https://betterstack.com/community/guides/scaling-go/golang-benchmarking/#comparing-benchmarks-with-benchstat